routes/web.php

```diff
@@ -1,18 +1,11 @@
 <?php
 
+use App\Http\Controllers\CityController;
 use Illuminate\Support\Facades\Route;
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider and all of them will
-| be assigned to the "web" middleware group. Make something great!
-|
-*/
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/cities', [CityController::class, 'index']);
+Route::post('/cities', [CityController::class, 'store'])->name('cities.store');
+Route::delete('/cities/{id}', [CityController::class, 'destroy'])->name('cities.destroy');
+Route::put('/cities/{id}', [CityController::class, 'update'])->name('cities.update');
+Route::get('/cities/json', [CityController::class, 'getCities'])->name('cities.json');
```

Comment:
You could group all the routes starting with 'cities' 

(https://laravel.com/docs/10.x/routing#route-group-prefixes)